tools/markdown-magic/src/transforms/packageImportInstructionsTransform.cjs

```diff
@@ -59,48 +71,29 @@ const generatePackageImportInstructionsSection = (
  * Generates a README section with instructions for how to import different API support levels based on
  * our standard package export paths (`/alpha`, `/beta`, `/legacy`).
  *
- * @remarks
- * The text content generated by this function assumes that export paths are used by the package.
- * If exports are not leveraged, use of this template is not recommended.
+ * Note: this function will only generate contents if one of our special export paths is found (`/alpha`, `/beta`, or `/legacy`).
+ * Otherwise, returns `undefined`.
```

Comment:
Cursory glance at the code suggests this to be the case?
```suggestion
 * Otherwise, returns an empty auto-generated section.
```

Reply:
Ah, I missed one. I think I'll just remove the second line here.
```suggestion
```